extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/S3TaskLogs.java

```diff
@@ -67,6 +67,11 @@ public S3TaskLogs(
   public Optional<InputStream> streamTaskLog(final String taskid, final long offset) throws IOException
   {
     final String taskKey = getTaskLogKey(taskid, "log");
+    // this is to satisfy CodeQL scan
+    Preconditions.checkArgument(
+        offset < Long.MAX_VALUE && offset > Long.MIN_VALUE,
```

Comment:
I think we can ignore the CodeQL scan for now because I am not entirely sure if Long.MIN_VALUE or Long.MAX_VALUE are not being used anywhere on purpose to represent some special scenarios.

The scan might also go away if we make the suggested change of adding a new method rather than updating the existing one. Although, I am not entirely sure if it will.

Reply:
ok, though PR can't be merged with CodeQL scan issue right?